descriptors/ModuleDescriptor-template.json

```diff
@@ -35,6 +35,18 @@
           "modulePermissions": [
             "modperms.inventory.items-in-transit-report.get"
           ]
+        },
+        {
```

Comment:
Hi @OleksandrVidinieiev, thank for updating the RAML file, could you please update the version in this file as well.

Reply:
I assume, this comment is irrelevant, since I extracted `pick-slips` interface definition into a separate file with version in both RAMLs being `0.1`.

Reply:
I assume, this comment is outdated, since I extracted `pick-slips` interface definition into a separate file with version in both RAMLs being `0.1`.